src/events/backup.py

```diff
@@ -30,7 +36,7 @@ class BackupEvents(Object):
 
     def __init__(self, charm):
         super().__init__(charm, "backup")
-        self.charm: "ZooKeeperCharm" = charm
+        self.charm: ZooKeeperCharm = charm
```

Comment:
**question:** ?

Reply:
Reverted in e1bc5ede55f3a2d55ee7dd6b2d4dae7830b960a3. I don't mind having the `__future__` import, so I did not pay attention when I changed that